examples/dreambooth/train_dreambooth.py

```diff
@@ -488,11 +498,14 @@ def get_full_repo_name(model_id: str, organization: Optional[str] = None, token:
 def main(args):
     logging_dir = Path(args.output_dir, args.logging_dir)
 
+    accelerator_project_config = ProjectConfiguration(total_limit=args.checkpointing_steps_total_limit)
```

Comment:
Nice! Didn't know about this. 

I always did the checkpointing logic manually with `accelerate` as shown in

https://github.com/huggingface/transformers/blob/9d1116e9951686f937d17697820117636bfc05a5/examples/pytorch/language-modeling/run_clm_no_trainer.py#L600

Reply:
@sayakpaul it's pretty new, it was first made available in `accelerate 0.16.0` I think: https://github.com/huggingface/accelerate/pull/916